### ISSUE TITLE
Caret Color Does Not Work

### ISSUE THREAD

**Issue by enchorb** [marklawlor/nativewind#668]
Following advice listed here: https://github.com/marklawlor/nativewind/issues/629

Placeholder color changes, caret color does not:

```
  <TextInput
          placeholder={'Placeholder'}
          className={
            'text-background-inverted caret:color-red-500 placeholder:text-green-400 h-10 w-full rounded-sm text-center'
          }
   />
```

**Comment by enchorb** [comment]
After more testing it does work on Android, not iOS. I'm assuming becuase Android uses `cursorColor` whereas iOS uses `selectionColor`

**Comment by marklawlor** [comment]
Yes `caret:` maps to `cursorColor` while `selection:` maps to `selectionColor`.

I think this is the most intuitive way to map it, but I am open to feedback.

**Comment by feledori** [comment]
Why deviate from cursorColor to caret? Wouldn't it be more trivial writing "cursor:"?

**Comment by marklawlor** [comment]
@feledori Because `cursor:` is already taken https://tailwindcss.com/docs/cursor

**Comment by marklawlor** [comment]
I just realised there was a typo in my original comment so I'd edited it.

https://tailwindcss.com/docs/caret-color -> `cursorColor`
https://tailwindcss.com/docs/hover-focus-and-other-states#highlighted-text -> `selectionColor`

**Comment by feledori** [comment]
@marklawlor And writing it like selection with ":" at the end so cursor:color-red-100 would be to similar?

**Comment by marklawlor** [comment]
I try to avoid creating a new className if an existing one exists. Diverging just creates future issues and more code to maintain 

Tailwind is configurable, so if you prefer that you can add it yourself as a plugin.
